### DISCUSSION TITLE
Add LinkedIn to site socials in admin/customization/config

### DISCUSSION THREAD

**Discussion by ellativity** [forem/forem#17096] [Feature Requests]
Several Forem Creators have been asking for a LinkedIn link/icon for the site socials list (to be displayed in the Nav links on the homepage) to be added to `admin/customization/config` > Social Media.

LinkedIn is increasingly being used for cross-posting and building followers for brands and communities, so it's as helpful to have a LinkedIn link there as a Facebook link. Discuss 😉

**Comment by ellativity** [comment]
https://github.com/forem/forem-internal-eng/issues/274
https://github.com/forem/forem-internal-eng/issues/322

**Comment by Kingston29142** [comment]
Yeah, you are risk. Linkedin is improving day by day. Thats why we have added it to the author box of [sharjah warriors ilt20 team](https://uaeplt20.com/sharjah-warriors-squad-for-ilt20-2023/). But now discussing it with the team so that we could developed a dedicated plugin which could insert the linkdin to the authors box.

**Comment by seisfebk** [comment]
Adding LinkedIn to your site's socials in the admin/customization/config section is a great way to connect with potential customers and partners on the professional social media platform. Here's a step-by-step guide on how to do it:

Log in to your website's admin panel.

Navigate to the customization or config section. This may be located in the settings or options menu.

Look for a section on social media links or icons. This is where you can add or edit the social media links that appear on your website.

If there is not a LinkedIn icon, you can add the link by creating a new social media icon or add the link to an existing icon.

Input the link to your LinkedIn profile or business page in the designated field.

Save your changes and check to see if the LinkedIn icon appears on your website.

Test the link to ensure that it is working correctly and directs users to your LinkedIn profile or business page.

Repeat these steps for any other social media platforms that you want to add to your website.

By adding a LinkedIn link to your website, you make it easy for users to connect with you on the platform and stay updated on your professional activities. Make sure that your profile is updated and represents your business in the best possible way.

Note: The exact steps to add the LinkedIn link may vary depending on the website platform or CMS that you are using. If you are unsure how to add the link, refer to the website's documentation or reach out to customer support for assistance.
Also, visit https://goread.io/blog/20-steps-for-launching-a-successful-instagram-influencer-marketing-campaign.
